lib/utils/zts_utils.dart

```diff
@@ -9,6 +9,8 @@ final List<Token> kDualCoin = <Token>[
   kQsrCoin,
 ];
 
+
+// TODO(maznnwell): these should match the coin details on the networks
```

Comment:
For both tokens, the name on the network matches the symbol. However, in the send token list, the `QSR` token is displayed as "Quesar," while the ZNN token is correctly displayed as `ZNN`.

Additionally, the token name is not unique. It is possible to create a token with the same name as another token but with a different symbol. Listing only the token name could lead to confusion, as it would not be clear which token is the correct one.

Reply:
From what I noticed, on production, for Zenon and Quasar, the token name and the symbol are the same: ZNN and QSR. When the user has balance in these two coins, the details from the production network is used. When the user doesn't have balance in these two coins, some hardcoded values are used: and in that case the names are Zenon and Quasar. 

I will change the hardcoded names to match those on the network.

The only thing that is unique about coins and tokens is the token standard. I could show a short version of the token standard next to the assets. But I could also use an icon to distinguish between a token and a coin.

Which one is best?

Reply:
The ZNN & QSR symbol names are validated by the SC. Having the same hardcoded definition as the network would solve the naming issue.

Considering the potential growth of tokens on the network, transitioning from a dropdown list to a search functionality is a logical step for scalability and usability, this is especially true for the receive dropbox.

For now, I think the native coins should always be at the top of the list or search result. Maybe even grouping them under a "Native Coins" section.

The tokens should be identified by a truncated token standard address. Although not exactly sure how this will fit in the dropbox.